src/app/components/IfAboveIE9Comment/index.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jam3/no-sanitizer-with-danger */
```

Comment:
Could be worth exploring another method of IE conditional logic that doesn't use `dangerouslySetInnerHTML`